pmm-tests/pmm-framework.sh

```diff
@@ -2566,37 +2566,40 @@ setup_mongodb_ssl () {
   echo "Setting up mongodb ssl"
   export PWD=$(pwd)
   setup_docker_compose
-  sudo mkdir -p /srv/pmm-qa || :
-  pushd /srv/pmm-qa
+  sudo mkdir -p /tmp/ssl || :
+  pushd /tmp/ssl
   if [ ! -d "pmm-ui-tests" ]; then
-    git clone https://github.com/percona/pmm-ui-tests
+    sudo git clone https://github.com/percona/pmm-ui-tests
```

Comment:
`sudo` for git clone ?

Reply:
sudo might not be needed for tmp I forgot to remove it, @IvanaBrecherova please remove sudo on mkdir and git clone, for /tmp